automation_tools/__init__.py

```diff
@@ -1622,6 +1623,30 @@ def setenforce(mode):
     run('setenforce {0}'.format(mode))
 
 
+def host_pings(host):
+    """This ensures the given IP/hostname pings succesfully.
+
+    :param host: A string. The IP or hostname of host.
+
+    """
+    if host is None:
+        print("The hostname/IP is not provided to ping. \
+        Terminating the execution !!")
+        sys.exit(1)
+    while True:
```

Comment:
@elyezer , Mean time for RHEL 7 is around 7-8 minutes and for RHEL 6 it is around 12-14 minutes.